lib/index.js

```diff
@@ -20,7 +23,7 @@ module.exports = function Conversation(bot, type) {
             var id = getId(msg);
             //If a dialog is currently open with this user, accept the message.
             debug('Checking if talking to   ' + id + ' = ' + !!_talkingTo[id]);
-            return _talkingTo[id];
+            return (customListener) ? _talkingTo[id] : false;
```

Comment:
In this line you may want to do `customListener()`. 

Right now, you're checking to see that the function is defined, rather than invoking it.